### DISCUSSION TITLE
Plugin videos de Vimeo

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#38] [Announcements]
# Plugin videos de Vimeo

El widget genera una lista de videos de Vimeo, también basado en el widget Lista de enlaces, mas la ayuda de un script muy pequeño escrito en vanilla js. Básicamente es igual que mi anterior aporte solo que con Vimeo en lugar de Youtube.

https://www.zkreations.com/2024/08/widget-lista-vimeo-lazyload.html

**Comment by TakoMics** [comment]
Muchas gracias por este aporte Daniel, este tipo de Widgets le dan mil años de vida a los blogs, aparte que en el punto estético, llenan de dinamismo la estructura de la web.

Mil gracias.

**Reply by danieIabel** [reply]
Me alegra que le sea de utilidad amigo, sali de vacaciones de la universidad y no he parado de programar cosas que voy a estar compartiendo, asi que estare muy activo!
